views/Chats.tsx

```diff
@@ -1,22 +1,62 @@
 import React, { Component } from 'react'
 import { View, Text, Image, StyleSheet, TextInput } from "react-native"
+import ChatList from "../components/ChatList"
+import axios from "axios"
+import getApiUrl from '../environment.js'
 import { connect } from "react-redux"
 
-export class Chats extends Component {
-    render() {
-        return (
-            <View style={styles.events}>
-                <Text>This is the chats component</Text>
-            </View>
-        )
-    }
+interface Props {
+  getChats: Function
+}
+
+export class Chats extends Component<Props> {
+  async componentDidMount() {
```

Comment:
Fetching chats. Right now it's hardcoded to fetch chats for "levans@email.com". This should of course be dynamic, but I wanted to focus on getting the data to create the markup and then refactor this part to fetch dynamic based on which user is logged in.